### PR TITLE
#411: Drop a few Node polyfills

### PR DESCRIPTION
**Pull request by fregante**
For #411 

Needs jq-web testing

### CODE REVIEW

File: browsers/webpack.config.js [hunk 1]
```diff
@@ -211,6 +211,10 @@ module.exports = (env, options) => ({
     },
     fallback: {
       fs: false,
+      crypto: false,
```
Comment (fregante) [inline browsers/webpack.config.js:214]: Loaded by

- jq-web, likely due to an automated ASM wrapper

File: browsers/webpack.config.js [hunk 2]
```diff
@@ -211,6 +211,10 @@ module.exports = (env, options) => ({
...
+      console: false,
```
Comment (fregante) [inline browsers/webpack.config.js:215]: Loaded by:

- unknown, no module uses `require('console')`

File: browsers/webpack.config.js [hunk 3]
```diff
@@ -211,6 +211,10 @@ module.exports = (env, options) => ({
...
+      vm: false,
```
Comment (fregante) [inline browsers/webpack.config.js:216]: Loaded by

- jsonpath-plus, with a check with its existence

File: browsers/webpack.config.js [hunk 4]
```diff
@@ -211,6 +211,10 @@ module.exports = (env, options) => ({
...
+      path: false,
```
Comment (fregante) [inline browsers/webpack.config.js:217]: Loaded by

- jq-web, likely due to an automated ASM wrapper

**Review by twschiller — APPROVED** [review]

**Pull request merged**